cmd/kubeadm/app/preflight/checks_windows.go

```diff
@@ -19,28 +19,37 @@ limitations under the License.
 package preflight
 
 import (
-	"os/exec"
-	"strings"
+	"os/user"
 
 	"github.com/pkg/errors"
 )
 
-// Check validates if an user has elevated (administrator) privileges.
+// The "Well-known SID" of Administrator group
+const administratorSID = "S-1-5-32-544"
```

Comment:
one minor request, could you please add this link above the constant too:
https://support.microsoft.com/en-us/help/243330/well-known-security-identifiers-in-windows-operating-systems

this is a standard practice in the go stdlib.

(commits amend should be force pushed)